src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts

```diff
@@ -108,6 +132,32 @@ export class HuiLightCardEditor extends LitElement
           .configValue="${"theme"}"
           @value-changed="${this._valueChanged}"
         ></hui-theme-select-editor>
+
+        <hui-action-editor
+          .label="${this.hass.localize(
+            "ui.panel.lovelace.editor.card.generic.hold_action"
+          )} (${this.hass.localize(
+            "ui.panel.lovelace.editor.card.config.optional"
+          )})"
+          .hass=${this.hass}
+          .config="${this._hold_action}"
+          .actions="${actions}"
+          .configValue="${"hold_action"}"
+          @action-changed="${this._valueChanged}"
+        ></hui-action-editor>
+
+        <hui-action-editor
+          .label="${this.hass.localize(
+            "ui.panel.lovelace.editor.card.generic.double_tap_action"
+          )} (${this.hass.localize(
+            "ui.panel.lovelace.editor.card.config.optional"
+          )})"
+          .hass=${this.hass}
+          .config="${this._double_tap_action}"
+          .actions="${actions}"
+          .configValue="${"double_tap_action"}"
+          @action-changed="${this._valueChanged}"
```

Comment:
And use standard events
```suggestion
          @value-changed="${this._valueChanged}"
```

Reply:
I can do that in another PR